README.md

```diff
@@ -4,6 +4,8 @@
 
 This project is developed and maintained by the [HAL team](https://github.com/rust-embedded/wg#the-hal-team).
 
+📣 `embedded-hal` v1.0 is now released! Check out the [announcement blog post](https://blog.rust-embedded.org/embedded-hal-v1/), the [API documentation](https://docs.rs/embedded-hal) and the [migration guide](docs/migrating-from-0.2-to-1.0.md).
```

Comment:
maybe you want to make this more useful using the new note/alert syntax from GH? (see [their blog post for more information](https://github.blog/changelog/2023-12-14-new-markdown-extension-alerts-provide-distinctive-styling-for-significant-content/))

Reply:
huh, nice. Styling looks nice, but it seems you can't remove the header saying "Note" or similar, so maybe it'd look a bit weird?

Reply:
i think labeling this as "important" or also "note" would be fine, including with the header?